### ISSUE TITLE
Go V1.7+ seems to be a requirement.

### ISSUE THREAD

**Issue by ymdatta** [praveenkumar/gopodman#6]
On Ubuntu16.04 using golang v1.6.2

 ```
 $go version
  go version go1.6.2 linux/amd64
 $go get github.com/praveenkumar/gopodman
  # github.com/praveenkumar/gopodman/vendor/github.com/projectatomic/libpod/cmd/podman/formats
src/github.com/praveenkumar/gopodman/vendor/github.com/projectatomic/libpod/cmd/podman/formats/formats.go:61: enc.SetIndent undefined (type *json.Encoder has no field or method SetIndent)
src/github.com/praveenkumar/gopodman/vendor/github.com/projectatomic/libpod/cmd/podman/formats/formats.go:63: enc.SetEscapeHTML undefined (type *json.Encoder has no field or method SetEscapeHTML)
src/github.com/praveenkumar/gopodman/vendor/github.com/projectatomic/libpod/cmd/podman/formats/templates.go:16: enc.SetEscapeHTML undefined (type *json.Encoder has no field or method SetEscapeHTML)
# github.com/praveenkumar/gopodman/vendor/github.com/varlink/go/varlink
src/github.com/praveenkumar/gopodman/vendor/github.com/varlink/go/varlink/service.go:248: l.(*net.UnixListener).SetUnlinkOnClose undefined (type *net.UnixListener has no field or method SetUnlinkOnClose)

```

According to this https://golang.org/doc/go1.7#minor_library_changes that enc.SetIndent has been introduced in go v 1.7.

I got stuck when trying to install gopodman, if it's true that it requires version 1.7+, then mentioning in the `README.md` would help.  :)

**Comment by praveenkumar** [comment]
@ymdatta Sure, I am doing mostly development >1.9, go ahead and create PR to update it.

**Comment by praveenkumar** [comment]
Closed by #7
